.github/workflows/rust.yml

```diff
@@ -94,31 +94,63 @@ jobs:
     strategy:
       matrix:
         source_os: [ubuntu-latest, windows-latest, local]
+    env:
+      api-level: 29
```

Comment:
I think it will be great to make `arch`, `profile` and `target` part of `env`.

Reply:
I didn't expect these to change as often as the `api-level` (which is also at least used in 3 places, the rest is only dereferenced twice), but it's indeed useful to deduplicate at least `emulator-options` which is a rather "complicated" string.

I really, really with GH starts supporting YAML anchors, it would have made this 100x better:
https://github.com/MarijnS95/android-ndk-rs/actions/runs/1494051637/workflow

Reply:
> I didn't expect these to change as often as the api-level

Maybe we should have `api-level` part of matrix? At least `min_sdk_version` and `target_sdk_version`.

Reply:
I guess now that we have this cache it's probably worth running it that many times?

Reply:
We can try :) But, probably, after we make sure that the CI is stable enough.

Reply:
However, we'll probably need to come up with a more interesting test/example too. All these versions do is limit API available at compile vs build time, and that's really only interesting if that API is actually used.

Otherwise the only thing we're checking is if the resulting APK is still compatible.

Reply:
Let's land such a change after the CI is stable like this (and it looks like it is already) - note that the examples default to `min_sdk_version=16`.